jsonnetfile.json

```diff
@@ -1,16 +1,24 @@
 {
-  "version": 1,
-  "dependencies": [
-    {
-      "source": {
-        "git": {
-          "remote": "https://github.com/openshift/cluster-monitoring-operator",
-          "subdir": "jsonnet"
+    "version": 1,
+    "dependencies": [
```

Comment:
I'm not sure if you intend to include this file in git if you don't want to compile.
You could add an ignore statement like
```yaml
.gitignore:
  additionalEntries:
    - jsonnetfile.json
```